packages/Uploader/server/upload.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
I understand this is not going to be the most high-traffic part of the repo, but requiring PHP to be installed, and also understanding PHP seems a maintenance liability.

Reply:
🤔 most mac has PHP installed automatically, so would not be a problem running it on macs, which is most of our cases currently. 
Also, I couldn't find a better alternative to mockup a simple server, handling files, while coding the uploader. 

Open to suggestions.

Reply:
no at this point. cypress can fake upload, from what I have read, but testing will happen in the next PR where I will be implementing the UI